pkg/metadata/provider_aws.go

```diff
@@ -101,7 +129,12 @@ func awsGet(url string) ([]byte, error) {
 		return nil, fmt.Errorf("AWS: http.NewRequest failed: %s", err)
 	}
 
-	resp, err := client.Do(req)
+	signedReq, err := signRequest(req, client)
+	if err != nil {
+		return nil, fmt.Errorf("AWS: Could not contact metadata service: %s", err)
```

Comment:
maybe make it a different error message than the one below, e.g. Could not sign metadata request

Reply:
Fixed